docs/docsite/rst/community/communication.rst

```diff
@@ -30,98 +30,51 @@ Request it in the `forum topic <https://forum.ansible.com/t/requesting-a-forum-g
-
-* an IRC client
-
-IRC chat supports:
-
-* no persistence (you only see messages when you are logged on unless you add a bouncer)
-* simple text interface
-* bridging from Matrix
-
-Our IRC channels may require you to register your IRC nickname. If you receive an error when you connect or when posting a message, see `libera.chat's Nickname Registration guide <https://libera.chat/guides/registration>`_ for instructions. To find all ``ansible`` specific channels on the libera.chat network, use the following command in your IRC client:
-
-.. code-block:: text
-
-   /msg alis LIST #ansible* -min 5
-
-as described in the `libera.chat docs <https://libera.chat/guides/findingchannels>`_.
-
-Our channels record history on the Matrix side. The channel history can be viewed in a browser - all channels will report an appropriate link to ``chat.ansible.im`` in their Chanserv entrymsg upon joining the room. Alternatively, a URL of the form ``https://chat.ansible.im/#/room/# {IRC channel name}:libera.chat`` will also work, for example -  for the #ansible-docs channel it would be `https://app.element.io/#/room/#ansible-docs:libera.chat`.
+.. note::
 
-.. _general_channels:
+  IRC channels are no longer considered official communication channels. Please use the Forum and Matrix instead.
 
 General channels
 ----------------
 
 The clickable links will take you directly to the relevant Matrix room in your browser; room/channel information is also given for use in other clients:
 
-- `Community social room and posting news for the Bullhorn newsletter <https://matrix.to:/#/#social:ansible.com>`_ - ``Matrix: #social:ansible.com | IRC: #ansible-social``
-- `General usage and support questions <https://matrix.to:/#/#users:ansible.com>`_ - ``Matrix: #users:ansible.com | IRC: #ansible``
-- `Discussions on developer topics and code related to features or bugs <https://matrix.to/#/#devel:ansible.com>`_ - ``Matrix: #devel:ansible.com | IRC: #ansible-devel``
-- `Discussions on community and collections related topics <https://matrix.to:/#/#community:ansible.com>`_ - ``Matrix: #community:ansible.com | IRC: #ansible-community``
-- `For public community meetings <https://matrix.to/#/#meeting:ansible.im>`_ - ``Matrix: #meeting:ansible.im | IRC: #ansible-meeting``
-   - We will generally announce these on one or more of the above mailing lists. See the `meeting schedule <https://github.com/ansible-community/meetings/blob/main/README.md>`_
+- `Community social room and posting news for the Bullhorn newsletter <https://matrix.to:/#/#social:ansible.com>`_
+- `General usage and support questions <https://matrix.to:/#/#users:ansible.com>`_
+- `Discussions on developer topics and code related to features or bugs <https://matrix.to/#/#devel:ansible.com>`_ 
+- `Discussions on community and collections related topics <https://matrix.to:/#/#community:ansible.com>`_
+- `For public community meetings <https://matrix.to/#/#meeting:ansible.im>`_
```

Comment:
Does anyone use this meeting channel? I didn't even know it existed in matrix land... :-)

Reply:
it's another ghost town, removed it, we should shut it down